### PR TITLE
[Term Entry] Dart: .elementAt()

### LINKED ISSUE #4296: [Term Entry] Dart List .elementAt()
### Reason/inspiration (optional)

We would like a new term entry under Dart [List](https://www.codecademy.com/resources/docs/dart/concept/list) for the Dart List `.elementAt` method. 
Provide the entry in a markdown file under `docs/content/dart/concept/list/terms/elementAt/elementAt.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - A syntax section with a pseudo code block with an explanation of it.
    - An example showing the method in use.

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Sriparno08**
### Description

- Added a new term entry for the .elementAt() method in Dart
- Closes #4296 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/list/terms/elementAt/elementAt.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Tags:
+  - 'Dart'
+  - 'Methods'
+  - 'Lists'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the `.elementAt()` method is used to access the element at a specific index in a list. The index needs to be a non-negative integer that's smaller than the length of the list.
```
Comment (avdhoottt) [inline content/dart/concepts/list/terms/elementAt/elementAt.md:16]:
```suggestion
In Dart, the **`.elementAt()`** method is used to access the element at a specific index in a list. The index needs to be a non-negative integer that's smaller than the length of the list.
```

File: content/dart/concepts/list/terms/elementAt/elementAt.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Here is an example that demonstrates the use of the `.elementAt()` method:
+
+```dart
+void main() {
+  List nums = ['11', '22', '33', '44', '55'];
+
+  // Using the .elementAt() method to access the element at nums[2]
+  var num = nums.elementAt(2);
+
```
Comment (avdhoottt) [inline content/dart/concepts/list/terms/elementAt/elementAt.md:37]:
```suggestion
```

File: content/dart/concepts/list/terms/elementAt/elementAt.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- `index`: The index that needs to be accessed.
+
+## Example
+
+Here is an example that demonstrates the use of the `.elementAt()` method:
+
+```dart
+void main() {
+  List nums = ['11', '22', '33', '44', '55'];
+
```
Comment (avdhoottt) [inline content/dart/concepts/list/terms/elementAt/elementAt.md:34]:
```suggestion
```

**Review by avdhoottt — APPROVED** [review]
Looks good for the second review!!

**Review by ishg-153 — APPROVED** [review]
@Sriparno08 Amazing Entry! Ready to merge.

Sorry for the delay🙃

**Pull request merged**